projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -191,15 +183,16 @@ class ProjectionRegistrySpec
       val allButTag001   = tagNames.tail
       val coordinates001 = WorkerCoordinates(projectionName, tagNames.head)
 
-      val testProbe = registerProjection(projectionName, tagNames.toSet)
-
+      val testProbe = TestProbe()
       // only let tagName001 messages pass
       testProbe.ignoreMsg {
         case FakeStarting(`tagName001`) => false
         case FakeStopping(`tagName001`) => false
         case _                          => true
       }
 
+      registerProjection(projectionName, tagNames.toSet, testProbe)
```

Comment:
We should create the `TestProbe` outside `registerProjection` and pass probe to it